sdk/eventhub/azure-eventhub-checkpointstoretable/setup.py

```diff
@@ -0,0 +1,76 @@
+from setuptools import setup, find_packages
+import os
+from io import open
+import re
+
+# example setup.py Feel free to copy the entire "azure-template" folder into a package folder named
+# with "azure-<yourpackagename>". Ensure that the below arguments to setup() are updated to reflect
+# your package.
+
+# this setup.py is set up in a specific way to keep the azure* and azure-mgmt-* namespaces WORKING all the way
+# up from python 2.7. Reference here: https://github.com/Azure/azure-sdk-for-python/wiki/Azure-packaging
+
+PACKAGE_NAME = "azure-eventhub-checkpointstoretable"
+PACKAGE_PPRINT_NAME = "Event Hubs checkpointer implementation with Tables Storage"
```

Comment:
Nit: The service has Table in the singular form
```suggestion
PACKAGE_PPRINT_NAME = "Event Hubs checkpointer implementation with Azure Table Storage"
```